x/lp/README.md

```diff
@@ -89,6 +91,24 @@ Expected format for `[swap-input-coins]` (omit curly brackets): "{denom0}{amount
 canined q estimate-swap-out "pool-name" [swap-input-coins] 
 ```
 
+#### list-records-from-pool
```

Comment:
Good call on the update here, reminded me I need to go through and redo some docs